archinstall/lib/disk/filesystem.py

```diff
@@ -181,31 +189,41 @@ def add_partition(self, partition_type :str, start :str, end :str, partition_for
 		else:
 			parted_string = f'{self.blockdevice.device} mkpart {partition_type} {start} {end}'
 
+		log(f"Adding partition using the following parted command: {parted_string}", level=logging.DEBUG)
+
 		if self.parted(parted_string):
 			count = 0
 			while count < 10:
 				new_uuid = None
 				new_uuid_set = (previous_partition_uuids ^ {partition.uuid for partition in self.blockdevice.partitions.values()})
+
 				if len(new_uuid_set) > 0:
 					new_uuid = new_uuid_set.pop()
+
 				if new_uuid:
 					try:
 						return self.blockdevice.get_partition(new_uuid)
 					except Exception as err:
-						print('Blockdevice:', self.blockdevice)
-						print('Partitions:', self.blockdevice.partitions)
-						print('Partition set:', new_uuid_set)
-						print('New UUID:', [new_uuid])
-						print('get_partition():', self.blockdevice.get_partition)
+						log(f'Blockdevice: {self.blockdevice}', level=logging.ERROR, fg="red")
+						log(f'Partitions: {self.blockdevice.partitions}', level=logging.ERROR, fg="red")
+						log(f'Partition set: {new_uuid_set}', level=logging.ERROR, fg="red")
+						log(f'New UUID: {[new_uuid]}', level=logging.ERROR, fg="red")
+						log(f'get_partition(): {self.blockdevice.get_partition}', level=logging.ERROR, fg="red")
 						raise err
 				else:
 					count += 1
-					log(f"Could not get UUID for partition. Waiting for the {count} time",level=logging.DEBUG)
+					log(f"Could not get UUID for partition. Waiting for the {count} time",level=logging.WARNING, fg="yellow")
```

Comment:
Can we make the text something like 'Retry attempt 1 of 10.' instead of Waiting for the {count} time"? It's more grammatically correct since we're not doing '1st', '2nd', etc.